src/app/pages/not-found/not-found.scss

```diff
@@ -0,0 +1,32 @@
+$spacing: 32px;
+
+main {
+  min-height: 100vh;
+  font-size: 1.25rem;
+}
+
+.wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: $spacing;
```

Comment:
I prefer a bit more padding on top (64px)
![Screen Shot 2021-01-11 at 21 00 00](https://user-images.githubusercontent.com/3506071/104260934-6bf1eb00-5452-11eb-8844-aaa910249d2a.png)

Versus the 32px used here:
![Screen Shot 2021-01-11 at 20 59 34](https://user-images.githubusercontent.com/3506071/104260936-6c8a8180-5452-11eb-9446-eeab1a36c9d3.png)